src/ildasm/dis.cpp

```diff
@@ -1646,7 +1646,7 @@ BOOL Disassemble(IMDInternalImport *pImport, BYTE *ILHeader, void *GUICookie, md
 
             case ShortInlineBrTarget:
             {
-                char offset = (char) pCode[PC];
+                signed char offset = (signed char) pCode[PC];
```

Comment:
Because offset is just used once here, I think we should just have:
long offset = pCode[PC];